bundles/org.openhab.binding.opensprinkler/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -4,444 +4,83 @@
 		xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
 		xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
 
-	<!-- The stand alone OpenSprinkler thing type -->
-	<thing-type id="http">
-		<label>OpenSprinkler</label>
-		<description>This is the stand alone OpenSprinkler device that allows sprinkler control.</description>
+    <bridge-type id="http">
```

Comment:
Please check formatting

Reply:
Applied the code style as installed with the IDE setup.